go/README.md

```diff
@@ -9,14 +9,20 @@ This folder contains most of the go code needed by the project.
 
 ## Download
 
-Main binary: `go get -u berty.tech/berty/v2/go/cmd/berty`
-
-Fetch library to your go cache: `go get -u berty.tech/berty/v2`
+Main binary:
+```sh
+git clone https://github.com/berty/berty
+cd berty/go
+make install
+
+# if things go wrong, please run and share the result of
+make doctor
```

Comment:
```suggestion
cd ..
make doctor
```
Or other such as a doctor rule in `go/Makefile` that would indicate where is actualy in the root of the repo, or even just calling `cd .. && make doctor`.

Reply:
`make doctor` from the go dir is also working

Reply:
```
moul@fwrz:~/go/src/berty.tech/berty/go (dev/moul/update-download-instructions) $ make doctor
go run ../tool/doctor/main.go
[+] OK     "sh" is present in $PATH (/bin/sh).
[+] OK     "make" is present in $PATH (/usr/bin/make).
[+] OK     "shasum" is present in $PATH (/usr/bin/shasum).
[+] OK     "go" is present in $PATH (/home/moul/.gvm/gos/go1.15.2/bin/go).
[+] OK     "yarn" is present in $PATH (/usr/bin/yarn).
[+] OK     "lsof" is present in $PATH (/usr/bin/lsof).
[+] OK     "adb" is present in $PATH (/usr/bin/adb).
[+] OK     "golangci-lint" is present in $PATH (/home/moul/go/bin/golangci-lint).
[+] OK     "docker" is present in $PATH (/usr/bin/docker).
[+] OK     valid go version: "1.15.2".
[+] OK     './vendor' directory does not exist.
```

Reply:
:ok_hand: my bad